packages/amplify-category-storage/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.js

```diff
@@ -287,13 +287,13 @@ function checkIfAuthExists(context) {
   return authExists;
 }
 
-function migrate(projectPath, resourceName) {
+function migrate(context, projectPath, resourceName) {
   const resourceDirPath = path.join(projectPath, 'amplify', 'backend', category, resourceName);
 
   // Change CFN file
 
   const cfnFilePath = path.join(resourceDirPath, 's3-cloudformation-template.json');
-  const oldCfn = JSON.parse(fs.readFileSync(cfnFilePath, 'utf8'));
+  const oldCfn = context.amplify.readJsonFile(cfnFilePath, 'utf8');
```

Comment:
By default, the encoding is 'utf8', so the second argument is not needed. 
The second argument is in the method just in case people wanted to use other encodings. 
I think we can consistently remove the second argument when readJsonFile method is called.